types.d.ts

```diff
@@ -248,9 +251,7 @@ export class Files {
         localDest?: boolean;
         noOverwrite?: boolean;
         progressCallback?: (...params: any[]) => any;
-    }): Promise<{
-        [key: string]: string;
-    }>;
+    }): Promise<{ key: string; }>;
```

Comment:
I'm not sure this means the return is a Promise that resolves to an array of objects? strange